tests/testClassification.py

```diff
@@ -75,6 +75,116 @@ def testAlgorithm(self):
             else:
                 self.assertEqual(probability, 1.0)
 
+    def testFlags(self):
+        """ Test all the failure modes of this algorithm, as well as checking that it succeeds when it should
```

Comment:
Technically, the coding standards insist this line should end with a ".".

I think the indentation of this docstring looks strange -- I can't find anything in the standards which says it's forbidden, but it's inconsistent with the other Python docstrings (e.g. in `meas.base.plugins`). I think you should consider reformatting it.